content/en/user/moderating.md

```diff
@@ -50,6 +50,10 @@ Filters normally apply to any status that contains the included characters, rega
 
 If you hide boosts from someone, you won’t see their boosts in your home feed. This option only appears on users who you are currently following.
 
+### Quote posts
+
+If your post is quoted by another user in an undesiredable fashion, you can [remove your quoted post](../quote-posts#remove).
```

Comment:
maybe "in an unwanted way" (which also fixes the typo'ed spelling)

Reply:
That was a truly odd thing to have written. Updated.